api/groups/baseAccountsGroup.go

```diff
@@ -137,7 +137,7 @@ func (group *accountsGroup) getAccounts(c *gin.Context) {
 		return
 	}
 
-	options, err := parseAccountQueryOptions(c)
+	options, err := parseAccountQueryOptions(c, addresses[0])
```

Comment:
I think this can panic if we did not provide any addresses

Reply:
added extra check